environments/quintupleo-routed.yaml

```diff
@@ -0,0 +1,68 @@
+parameter_defaults:
```

Comment:
Environments in ovb should be generated based on https://github.com/cybertron/openstack-virtual-baremetal/blob/master/sample-env-generator/environments.yaml